### PR TITLE
Make tmp variables const

### PR DESCRIPTION
**Pull request by ghost**
Many times, when we create a tmp variable for the purpose of reassigning it to the variable we got it from, the variable is not changed at all or must be protected from any change at all. For this reason, it makes sense to make those tmp variables const, to ensure they are not changed by any side effects of any other function.

### CODE REVIEW

File: src/libraries/System.DirectoryServices/src/System/DirectoryServices/ActiveDirectory/RoleOwnerCollection.cs [hunk 1]
```diff
@@ -30,7 +30,7 @@ public bool Contains(ActiveDirectoryRole role)
 
             for (int i = 0; i < InnerList.Count; i++)
             {
-                int tmp = (int)InnerList[i];
+                const int tmp = (int)InnerList[i];
```
Comment (stephentoub) [inline src/libraries/System.DirectoryServices/src/System/DirectoryServices/ActiveDirectory/RoleOwnerCollection.cs:33]: The changes in this file will not compile.

File: src/libraries/System.Drawing.Common/src/System/Drawing/Printing/PageSettings.Unix.cs [hunk 2]
```diff
@@ -97,7 +97,7 @@ public Rectangle Bounds
                 if (this.landscape)
                 {
                     // swap width and height
-                    int tmp = width;
+                    const int tmp = width;
```
Comment (stephentoub) [inline src/libraries/System.Drawing.Common/src/System/Drawing/Printing/PageSettings.Unix.cs:100]: The changes in this file will not compile.

File: src/libraries/System.Collections/src/System/Collections/Generic/Queue.cs [hunk 3]
```diff
@@ -364,7 +364,7 @@ private void MoveNext(ref int index)
             // It is tempting to use the remainder operator here but it is actually much slower
             // than a simple comparison and a rarely taken branch.
             // JIT produces better code than with ternary operator ?:
-            int tmp = index + 1;
+            const int tmp = index + 1;
```
Comment (stephentoub) [inline src/libraries/System.Collections/src/System/Collections/Generic/Queue.cs:367]: The changes in this file will not compile.

**Review by stephentoub — COMMENTED** [review]
These will still not compile. Please build your changes locally before pushing them up. Thanks.

**Comment by pi1024e** [comment]
Done @stephentoub, sorry about that.

**Comment by pi1024e** [comment]
@stephentoub any updates?

**Comment by BrzVlad** [comment]
I don't see any benefit to having const everywhere. I actually dislike using const because it just adds code bloat when it's obvious from the context that the variable doesn't change in value. In the end it's just a matter of personal preference and I don't think we should just change random code due to a personal preference.

**Comment by stephentoub** [comment]
Thanks for the PR.  Closing due to the feedback.

**Pull request closed without merging**